BookStoreProd/src/main/java/mate/academy/bookstoreprod/model/Book.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.bookstoreprod.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+@Entity
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @NotNull
```

Comment:
We use `@NotNull` and similar annotations for DTO validation from HTTP request, for entities you have `@Column` annotation with different properties: nullable, unique etc
```suggestion
    @Column(nullable = false)
```